sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Azure.Messaging.EventHubs.Processor.Tests.csproj

```diff
@@ -25,13 +25,12 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core\tests\TestFramework\Azure.Core.TestFramework.csproj" />
```

Comment:
Should we add a property in the engineering system to abstract away this path?   Something like [`AzureCoreSharedSources`](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Directory.Build.Data.props#L122)?